### ISSUE TITLE
Failed to set ntp: NTP not supported

### ISSUE THREAD

**Issue by lsiem** [berndhofer/freqstart#51]
When running the script (**develop** branch) with the setup routine, the following error occurs:

```
Failed to set ntp: NTP not supported
Error in freqstart.sh in function _fsPkgs_ on line 2872
```

**Comment by berndhofer** [comment]
I maybe kick out that check. Which vps wouldnt have setup timesync anyway.

**Comment by berndhofer** [comment]
Simplified the routine and only use chrony commands. Fixed in v1.0.3
